ledger/heavy/handler/handler.go

```diff
@@ -398,3 +407,37 @@ func (h *Handler) handleHeavyPayload(ctx context.Context, genericMsg insolar.Par
 
 	return &reply.OK{}, nil
 }
+
+func (h *Handler) cloneUpToLatest(ctx context.Context, from insolar.PulseNumber) {
+	logger := inslogger.FromContext(ctx)
+	latest, err := h.PulseAccessor.Latest(ctx)
+	if err != nil {
+		logger.Error(errors.Wrapf(err, "failed to get latest pulse"))
+		return
+	}
+	next, err := h.PulseCalculator.Forwards(ctx, from, 1)
+	if err == pulse.ErrNotFound {
+		return
+	}
+	if err != nil {
+		logger.Error(errors.Wrapf(err, "failed to get next pulse number"))
+		return
+	}
+	for to := next.PulseNumber; to <= latest.PulseNumber; {
+		err := h.JetModifier.Clone(ctx, from, to)
```

Comment:
This is wrong. We should never clone tree and override existing trees.

Reply:
If the drop can't be late, then I'll remove the excessive cloning. Is it?